methcomp/comparer.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+
+"""Abstract comparer base class
+"""
+from typing import Any, Dict
+from abc import ABC, abstractmethod
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+
+
+class Comparer(ABC):
+
+    """Abstract method comparison base class
+
+    Attributes
+    ----------
+    calculated : bool
+        True if result is calculated
+    method1 : np.ndarray
+        Values for method 1
+    method2 : np.ndarray
+        Values for method 2
+    n : int
+        Length of method value vectors
+    result : dict
+        Calculation result
+    """
+
+    def __init__(self, method1: np.ndarray, method2: np.ndarray):
```

Comment:
didn't we define method1/method2 as Union[List[float], np.ndarray]?

Reply:
Agreed, I changed this now in Comparer, Regressor, Linear, Deming, & PassingBablok.

Reply:
resolved.